README.md

```diff
@@ -1,7 +1,7 @@
 # KPI
 
 [![Python Build Status](https://github.com/kobotoolbox/kpi/workflows/pytest/badge.svg)](https://github.com/kobotoolbox/kpi/actions?query=workflow%3Apytest)
-[![Python Coverage Status](https://coveralls.io/repos/github/kobotoolbox/kpi/badge.svg?branch=master)](https://coveralls.io/github/kobotoolbox/kpi?branch=master)
+[![Python Coverage Status](https://coveralls.io/repos/github/kobotoolbox/kpi/badge.svg?branch=main)](https://coveralls.io/github/kobotoolbox/kpi?branch=main)
 [![JavaScript Build Status](https://github.com/kobotoolbox/kpi/workflows/npm-test/badge.svg)](https://github.com/kobotoolbox/kpi/actions?query=workflow%3Anpm-test)
 
```

Comment:
for this PR not sure if it's in scope, but it'd be cool to have this infoin the readme:

- "main is the development branch" spiel [[xref 🗨️ internal chat](https://chat.kobotoolbox.org/#narrow/stream/4-Kobo-Dev/topic/.60beta.60.20branch.20is.20unnecessary/near/460121)]
- convenience one-liner for switching to the latest release branch [[📄 here too](https://github.com/kobotoolbox/kobo-install/pull/244#discussion_r1781301987)]